src/routes/our-music/index.svelte

```diff
@@ -65,10 +65,10 @@
     .iframe-container-16x9 {
         padding-bottom: calc(9 / 16 * 100%);
     }
-    .caption {
+    /* .caption {
```

Comment:
In general, it's a bad practice to keep the unused code as comments. Most likely we'll never use that again. So, let's remove all commented codes for now. In the worst case, we can always check git history for the code.